packages/core/src/amazonqFeatureDev/session/session.ts

```diff
@@ -84,6 +91,8 @@ export class Session {
             span.record({ amazonqConversationId: this._conversationId, credentialStartUrl: AuthUtil.instance.startUrl })
         })
 
+        await this.showDevPortal(msg)
```

Comment:
This PR will be discarded.

```suggestion
        ## No longer needed
        await this.showDevPortal(msg)
```